### DISCUSSION TITLE
📺 普林斯顿 COS126 计算机科学：跨学科方法 ⭐️ | HackWay技术学习路线

### DISCUSSION THREAD

**Discussion by giscus** [lidongyx/hackwaydoc#55] [Announcements]
# 📺 普林斯顿 COS126 计算机科学：跨学科方法 ⭐️ | HackWay技术学习路线

课程名称： COS126 Computer Science: An Interdisciplinary Approach

https://hackway.org/docs/cs/freshman/first/cos126

**Comment by DanielDanyang** [comment]
官网貌似要求学生的账号，进不去

**Reply by lidongyx** [reply]
以前是不需要的，服了，现在美国很多名校的课程都不怎么对外开放了，我去把它的星号去掉。

**Reply by fan-tastic-z** [reply]
https://www.cs.princeton.edu/courses/archive/spr24/cos126/
24年的目前是可以打开的

**Comment by zhangbird050801** [comment]
除了这个还有教学 java 的课程吗

**Reply by fan-tastic-z** [reply]
https://www.cs.princeton.edu/courses/archive/spr24/cos126/
24年的目前是可以打开的

**Reply by itZONK** [reply]
打开也没用啊，只有ppt不说videos了，连个reading都没得

**Comment by LordBao666** [comment]
你猜怎么着？Princeton在 Coursera 有对应开设的课程
1 Computer Science: Programming with a Purpose
2 Computer Science: Algorithms, Theory, and Machines
